src/core_plugins/kibana/public/management/sections/objects/_view.html

```diff
@@ -1,36 +1,127 @@
-<kbn-management-app section="kibana">
-  <kbn-management-objects-view class="container">
-    <div class="pull-right" style="margin-top: 20px;">
-      <a href="{{ link }}" class="btn btn-primary">View {{ title }}</a>
-      <a confirm-click="delete()" class="btn btn-danger"><i class="fa fa-trash-o"></i> Delete {{ title }} Object</a>
+<kbn-management-app section="kibana" class="kuiView">
+  <kbn-management-objects-view class="kuiViewContent kuiViewContent--constrainedWidth">
+    <!-- Header -->
+    <div class="kuiViewContentItem kuiSubHeader">
+      <h1 class="kuiTitle">
+        Edit {{ title }}
+      </h1>
+
+      <div class="kuiButtonGroup">
+        <a
+          class="kuiButton kuiButton--basic kuiButton--iconText"
+          href="{{ link }}"
+        >
```

Comment:
I totally agree that the vertical listing of attributes makes sense, and 100% support that. I really  don't like the floating `>` of the opening tag...

> ...Conceptually, this makes sense to me too, since the enclosing tags become a parent node and the text content becomes a child node. 
> For example, I might scan and think content is an attribute if the closing bracket is on the same line

I conceptualize it in the opposite way. This really only becomes an issue when a tag contains simple text as its content. In that case, I see the attributes of a tag and the contents of a tag to all describe what that tag is supposed to represent, and think it makes sense to have them all justified.

The `>` isn't a 'tag' it's part of the definition of the opening tag of the element.

> This might only make sense to me, but I also think of this way of indenting brackets as similar to the way you would indent the opening and closing braces and brackets in object and array definitions. 

You pattern if applied to object and array declaration would be more akin to this, which for clarity, I would not suggest.
```
const object = {
  'a':
  {
  },
  'b': 
  [
    0,
    1,
    2,
  ],
};
```